components/gantt/gantt-tree/filter/overview.md

```diff
@@ -22,8 +24,18 @@ The behavior of the filter input and the available filter operators will depend
 
 You can filter more than one column at a time, and all filter rules will be applied together with an `AND` logic.
 
+## Filter Descriptors
+
+The Gantt filter state is stored in [CompositeFilterDescriptors](https://docs.telerik.com/blazor-ui/api/Telerik.DataSource.CompositeFilterDescriptor). The below information is important if you want to [get or change the Gantt filters programmatically]({%slug gantt-state%}).
+
+Each `CompositeFilterDescriptor` contains a [**collection** of `FilterDescriptor`s](https://docs.telerik.com/blazor-ui/api/Telerik.DataSource.FilterDescriptorCollection) which can target the same field or different fields. All descriptors in the collection are applied with an *AND* or an *OR* `LogicalOperator`.
```

Comment:
I am not 100% sure about this, but normally, one CompositeFilterDescriptor targets only one field (probably the Search descriptor is the only exception). So do we want to mention "or different fields" ?